kitchen-sink/src/views/MainPanel.js

```diff
@@ -28,9 +28,9 @@ const MainPanel = kind({
 
 	render: (props) => (
 		<Panel {...props}>
-			<Divider spacing="normal" startSection>
+			<Heading startSection>
```

Comment:
I'm wondering if all the other changes to `Divider` with no props should be `<Heading size="small">` or (`large`)?